debian/entry.sh

```diff
@@ -65,16 +71,18 @@ function init_non_systemd()
 	# trap the stop signal then send SIGTERM to user processes
 	trap signal_handler SIGRTMIN+3 SIGTERM
 
-	udevd & 
-	udevadm trigger &> /dev/null
-	
 	CMD=$(which "$1")
 	# echo error message, when executable file doesn't exist.
 	if [ $? == '0' ]; then
 		shift
+		if [ ! -z "$RESIN" ] && [ ! -z "$RESIN_DEVICE_UUID" ]; then
+			# run this on resin device only
+			start_udev
+		fi
 		"$CMD" "$@" &
 		pid=$!
 		wait $pid
+		fg &> /dev/null
```

Comment:
Why are you redirecting output to /dev/null? You want the stdout/stderr streams of this process to be attached to whoever started the container. Being an interactive shell or the docker log management